src/main/scala/tile/BaseTile.scala

```diff
@@ -72,12 +72,13 @@ trait HasNonDiplomaticTileParameters {
 
   // TODO merge with isaString in CSR.scala
```

Comment:
Does "isaString in CSR.scala" do the right thing?

Reply:
No, and it didn't matter for what I was trying to do, but I see no reason not to fix it.